add multiple fake units functionality

Ridepad opened the pull request:
https://user-images.githubusercontent.com/24712455/229607096-349965ea-8b77-4ec1-a9a7-b061e84054a3.mp4

bkader commented:
Hello @Ridepad and sorry for my late reply.
Can you share an example of how to add custom unit in the `Tables.lua` file? Just to understand how your changes work.
Thanks in advance!

Ridepad commented:
Same way it worked before and now can be table of tables:
https://github.com/bkader/Skada-WoTLK/pull/108/commits/cac8bb00eee57ae588fcb8c3d428e5654be18952#diff-38968cbd83c5ae4ca0773c8a09024b41584e9682dcf09fcfc333811311e5d136R161


		-- ICC: Professor Putricide
		[36678] = {
			{
				text = "%s - Phase 1",
				stop = 0.8,
			},
			{
				text = "%s - Phase 2",
				start = 0.8,
				stop = 0.35
			},
			{
				text = "%s - Phase 3",
				start = 0.35,
			},
			{
				text = "5%% before p2",
				start = 0.85,
				stop = 0.8,
			},
			{
				text = "2%% before p2",
				start = 0.82,
				stop = 0.8,
			},
			{
				text = "5%% before p2",
				start = 0.4,
				stop = 0.35,
			},
			{
				text = "2%% before p2",
				start = 0.37,
				stop = 0.35,
			},
		},

		-- ICC: Sindragosa
		[36853] = {
			text = L["%s - Phase 2"],
			start = 0.35,
		},

Ridepad commented:
After testing for a month, now should work properly.
Added Tables.lua, changed description about custom units, added some
Also fixed bug on LK when smart stopped on FoF and periodic damage keeps creating new segments.
[compare](https://github.com/bkader/Skada-WoTLK/compare/main...Ridepad:Skada-WoTLK:main)
